reviewers.md

```diff
@@ -0,0 +1,94 @@
+| linux                        | Volodymyr Babchuk | Oleksandr Tyshchenko |
+| meta-xt-common               | Volodymyr Babchuk | Ruslan Shymkevych    |
+| meta-xt-prod-devel-rcar      | Volodymyr Babchuk | Ruslan Shymkevych    |
+| meta-xt-prod-devel-rcar-gen4 | Volodymyr Babchuk |                      |
+| meta-xt-rcar                 | Volodymyr Babchuk | Ruslan Shymkevych    |
+| moulin                       | Volodymyr Babchuk | Ruslan Shymkevych    |
+| u-boot                       | Volodymyr Babchuk | Oleksandr Tyshchenko |
+| xen                          | Volodymyr Babchuk | Oleksandr Tyshchenko |
+
+## Group A
+
+It's like 'Group A+' but without plus.
+| Repo                            | Reviewer             | Reviewer          |
+|---------------------------------|----------------------|-------------------|
+| android_device_xenvm            | Andrii Chepurnyi     | Ruslan Shymkevych |
+| android_external_drm_hwcomposer | Andrii Chepurnyi     | Ruslan Shymkevych |
+| android_kernel_manifest         | Andrii Chepurnyi     | Ruslan Shymkevych |
+| android_manifest                | Andrii Chepurnyi     | Ruslan Shymkevych |
+| android_vendor_xen_audio        | Andrii Chepurnyi     | Ruslan Shymkevych |
+| android_vendor_xen_gnss         | Andrii Chepurnyi     | Ruslan Shymkevych |
+| android_vendor_xen_sensors      | Andrii Chepurnyi     | Ruslan Shymkevych |
+| android_vendor_xen_vehicle      | Andrii Chepurnyi     | Ruslan Shymkevych |
+| camera_be                       | Andrii Chepurnyi     | Oleksandr Grytsov |
+| DisplayManager                  | Andrii Chepurnyi     | Oleksandr Grytsov |
+| displ_be                        | Andrii Chepurnyi     | Oleksandr Grytsov |
+| kmgk                            | Andrii Chepurnyi     | Ruslan Shymkevych |
+| libvisclient                    | Andrii Chepurnyi     |                   |
+| libxenbe                        | Andrii Chepurnyi     |                   |
+| moulin-rcar-gen3                | Volodymyr Babchuk    |                   |
+| optee_os                        | Volodymyr Babchuk    | Andrii Chepurnyi  |
+| qemu                            |                      |                   |
+| snd_be                          | Oleksandr Grytsov    |                   |
+| uWebSockets                     | Andrii Chepurnyi     |                   |
+| virtio-disk                     | Oleksandr Tyshchenko |                   |
+| xt-log                          | Oleksandr Grytsov    |                   |
+
+## Previous generation projects
+| Repo                      | Reviewer          | Reviewer |
+|---------------------------|-------------------|----------|
+| build-scripts             | Ruslan Shymkevych |          |
+| manifests                 | Ruslan Shymkevych |          |
+| meta-xt-agl-base          | Ruslan Shymkevych |          |
+| meta-xt-images            | Ruslan Shymkevych |          |
+| meta-xt-prod-aos          | Ruslan Shymkevych |          |
```

Comment:
Put me here as well. Also, we close support of this product. I mean new features will not be added there. We have
created new product based on moulin: https://github.com/xen-troops/meta-xt-prod-aos-rcar. Please add it. If you would like you can be reviewer at this repo as well. But also add me.

Reply:
Added as reviewer to meta-xt-prod-aos.
Repo meta-xt-prod-aos-rcar is added with you as reviewer.